src/main/java/com/sixt/service/framework/rpc/HttpClientWrapper.java

```diff
@@ -108,12 +109,16 @@ public ContentResponse execute(HttpRequestWrapper request, RpcCallExceptionDecod
                     if (orangeContext != null) {
                         spanContext = orangeContext.getTracingContext();
                     }
+                    String serviceMethod = String.format("%s.%s", client.getServiceName(), client.getMethodName());
```

Comment:
please change to simple string concat, it's lighter-weight, and with java 8, strings are in the normal heap pool also.  maybe even just make it a method in the RpcClient class